src/upload_data.py

```diff
@@ -0,0 +1,23 @@
+import boto3
+import requests
+import argparse
+import os
+
+cwd = os.getcwd()
+
+data_permanent_url = 'http://s3.us-east-2.amazonaws.com/nw-carsonchen-acedata/atp_data.csv'
+file_name = "atp_data.csv"
+
```

Comment:
It would nice to have a few logging messages to confirm that the data load is succesful.

Reply:
Thanks for the suggestion. This has been added.

Reply:
Thanks! I see the individual scripts have logging messages, however the scripts do not have a logger configured, so I think they will not display.
You might consider defining a logger in the src/upload_data.py as :
import logging.config
logging.config.fileConfig(config.LOGGING_CONFIG)

Reply:
Good catch. I configured a logger that saves.